addons/website_slides/static/src/js/slides_course_fullscreen_player.js

```diff
@@ -775,6 +660,20 @@
             ev.preventDefault();
             this._toggleSidebar();
         },
+
+        _onClickShareSlide: function (ev) {
+            const slide = this.get('slide');
+            this.call("dialog", "add", SlideShareDialog, {
+                category: slide.category,
+                emailSharing: slide.emailSharing === 'True',
+                embedCode: slide.embedCode || false,
```

Comment:
```suggestion
                embedCode: slide.embedCode || '',
```

Since embedCode is supposed to be a string, do this instead to follow the validation you gave to the Component.